pkg/apis/serving/v1/revision_validation.go

```diff
@@ -182,18 +182,16 @@ func validateQueueSidecarAnnotation(annotations map[string]string) *apis.FieldEr
 	if len(annotations) == 0 {
 		return nil
 	}
-	v, ok := annotations[serving.QueueSideCarResourcePercentageAnnotation]
+	k, v, ok := serving.QueueSidecarResourcePercentageAnnotation.Get(annotations)
 	if !ok {
```

Comment:
another nit: should we use `m` instead of `annotations` for the parameter name for consistency with `annotation_validation.go`?